File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.java
class Solution {
    public String solution(String my_string) {
        String answer = "";
        
        for(int i=0;i<my_string.length();i++){
            char item = my_string.charAt(i);
            if(Character.isUpperCase(item)){
                answer += Character.toLowerCase(item);
            }else{
                answer += Character.toUpperCase(item);
            }
        } 
        return answer;
    }
}

File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

char* solution(const char* my_string) {
    char* answer = (char*)malloc(strlen(my_string) +1);
    int i;

    for(i=0;i<strlen(my_string);i++){    
        if(my_string[i] >= 'A' && my_string[i] <= 'Z') answer[i] = my_string[i] + 32;
        if(my_string[i] >= 'a' && my_string[i] <= 'z') answer[i] = my_string[i] - 32;
      
    }
    
    answer[i] = '\0';
    return answer;
}